src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleCRUDAnonTests.java

```diff
@@ -57,37 +55,48 @@ public void ArticleCRUDAnon_003_addDropdown() {
 
   @Test(dataProviderClass = ArticleDataProvider.class, dataProvider = "articleTitles",
       groups = {"ArticleCRUDAnon_004"})
-  @RelatedIssue(issueID = "MAIN-6278", comment = "test manually")
-  public void ArticleCRUDAnon_004_differentTitles(String articleTitle) {
+  public void articleCRUDAnon_differentTitles(String articleTitle) {
     String articleContent = PageContent.ARTICLE_TEXT;
     String randomArticleTitle = articleTitle + DateTime.now().getMillis();
     VisualEditModePageObject visualEditMode =
         new ArticlePageObject().navigateToArticleEditPage(wikiURL, randomArticleTitle);
     visualEditMode.addContent(articleContent);
-    ArticlePageObject article = visualEditMode.submitArticle();
+    ArticlePageObject article = visualEditMode.submitExpectingNotification().submitArticle();
     article.verifyContent(articleContent);
     article.verifyArticleTitle(randomArticleTitle);
   }
 
   @Test(groups = {"ArticleCRUDAnon_005"})
-  @RelatedIssue(issueID = "MAIN-6278", comment = "test manually")
-  public void ArticleCRUDAnon_005_editByURL() {
+  public void articleCRUDAnon_editByURL() {
     String articleContent = PageContent.ARTICLE_TEXT;
     ArticlePageObject article = new ArticlePageObject().open("AnonEditByURL");
     VisualEditModePageObject visualEditMode = article.navigateToArticleEditPage();
     visualEditMode.addContent(articleContent);
-    visualEditMode.submitArticle();
+    visualEditMode.submitExpectingNotification().submitArticle();
     article.verifyContent(articleContent);
   }
 
   @Test(groups = {"ArticleCRUDAnon_006"})
-  @RelatedIssue(issueID = "MAIN-6278", comment = "test manually")
-  public void ArticleCRUDAnon_006_editDropdown() {
+  public void articleCRUDAnon_editDropdown() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)